velox/exec/Spiller.cpp

```diff
@@ -569,6 +569,24 @@ std::string Spiller::toString() const {
       spillFinalized_);
 }
 
+int32_t Spiller::Config::spillLevel(uint8_t startBitOffset) const {
+  VELOX_CHECK_LE(startBitOffset + hashBitRange.numBits(), 64);
```

Comment:
nit: perhaps, include startBitOffset and hashBitRange.numBits() in the error message for easier troubleshooting